addons/account/models/account.py

```diff
@@ -589,8 +589,8 @@ def _get_bank_statements_available_sources(self):
              "SEPA Credit Transfer: Pay bill from a SEPA Credit Transfer file you submit to your bank. Enable this option from the settings.")
     at_least_one_inbound = fields.Boolean(compute='_methods_compute', store=True)
     at_least_one_outbound = fields.Boolean(compute='_methods_compute', store=True)
-    profit_account_id = fields.Many2one('account.account', string='Profit Account', domain=[('deprecated', '=', False)], help="Used to register a profit when the ending balance of a cash register differs from what the system computes")
```

Comment:
@rhe-odoo , Yop I've just talked to AL, because @caburj spotted a problem with properties. We shouldn't have listen to lap and accounting_team, and we should anly put default values on the company and use those as default on journals.

The problem with the properties, is they are different by companies, and the journal always have one and only one company. So it means that if a user is in a company different than the of the journal, it will show the account in property of the company of the user, and not the correct account which is in the company of the journal.

Sorry.